src/pydocstyle/config.py

```diff
@@ -139,31 +141,42 @@ def _get_matches(config):
             match_dir_func = re(config.match_dir + '$').match
             return match_func, match_dir_func
 
+        def _get_ignore_dec(config):
```

Comment:
I prefer if you use the full word `decorator`, as this is not a common shorthand (like `func`).